src/query/storage/converter.go

```diff
@@ -278,28 +278,32 @@ func iteratorToTsSeries(
 	iter encoding.SeriesIterator,
 	enforcer cost.ChainedEnforcer,
 	tagOptions models.TagOptions,
-) (*ts.Series, error) {
+) (*ts.Series, *ts.Exemplars, error) {
 	metric, err := FromM3IdentToMetric(iter.ID(), iter.Tags(), tagOptions)
 	if err != nil {
-		return nil, err
+		return nil, nil, err
 	}
 
 	datapoints := make(ts.Datapoints, 0, initRawFetchAllocSize)
+	exemplars := make(ts.Exemplars, 0)
```

Comment:
nit: init with `initRawFetchAllocSize` (or add a `Remaining()` method to encoding.SeriesIterator`?